composer.json

```diff
@@ -52,5 +52,8 @@
 		"forum": "http://forum.codeigniter.com/",
 		"source": "https://github.com/codeigniter4/CodeIgniter4",
 		"slack": "https://codeigniterchat.slack.com"
-	}
+	},
+	"suggest": {
+ 		"ext-fileinfo": "Improves mime type detection for files"
+  }
```

Comment:
Please align the closing brace here. Then move the suggests section after require-dev. Add this also to admin/starter/composer.json

Reply:
What about `admin/module` and `admin/framework?`

Reply:
Ah yes, forgot those two. Kindly change them too.